components/filters/FilterPicker.tsx

```diff
@@ -0,0 +1,187 @@
+import { DataList } from "@/components/DataList";
+import { Filter } from "@/lib/api";
+import { useWindowWidth } from "@/lib/hooks";
+import { useLayoutEffect, useRef, useState } from "react";
+import theme from "styles/theme";
+import Selectable, { SelectableSize } from "../form/Selectable";
+import FilterPickerCategory from "./FilterPickerCategory";
+
+export interface PickerFilter extends Filter {
+  active?: boolean;
+}
+
+interface FilterPickerProps {
+  filtersList: PickerFilter[];
+  activeFilters: PickerFilter[];
+  onFilterClick: (id?: string, filterType?: string) => any;
+  onFilterSeclect: (filterSelect?: string, enable?: boolean) => any;
+  selectedMemberCount?: number;
+}
+
+export default function FilterPicker({
+  filtersList,
+  activeFilters,
+  onFilterClick,
+  onFilterSeclect,
+  selectedMemberCount,
+}: FilterPickerProps) {
+  const width = useWindowWidth();
+  const [defaultHeight, setDefaultHeight] = useState<number>();
+  const [focusActive, setFocusActive] = useState<boolean>(true);
+  const [industryActive, setIndustryActive] = useState<boolean>();
+  const [regionActive, setRegionActive] = useState<boolean>();
+  const [experienceActive, setExperienceActive] = useState<boolean>();
+  const listRef = useRef<HTMLUListElement>();
+  const listItemsRef = useRef<HTMLLIElement[]>([]);
+  const filterIsSelected = activeFilters.length !== 0;
+
+  useLayoutEffect(() => {
+    if (!listItemsRef) return;
+    setDefaultHeight(listRef.current.scrollHeight);
+  }, [width]);
+
+  function activateFilter(
```

Comment:
My gut is that these look and feel like tabs, but don't behave like tabs. What do you think about adjusting the behavior to match how tabs more typically behave?

![gif](https://user-images.githubusercontent.com/5141111/195272342-b4315f8c-9029-4615-afd9-e1db15c83353.gif)

This means:
- **One (and only one) can be selected at a time.** Not more than one or zero.
- **Clicking on one deselects the others.** The only exception is when you click the active tab (which does nothing).

Reply:
I agree I like the tab selection, it actually also in theory keeps the user knowing what category of filter they're looking at. I'll add this to the next commit.

Also to your point Andrew, would this tab behavior make it more clear for filter selection? I think your tooltip suggestion would have been most helpful for the multiple filter categories being shown at once but since we're all in agreeance that tab behavior is better I think that solves it? Or maybe I'm not seeing some other benefits you're seeing.

Reply:
update is live here: https://hawaiiansintech-53p35qftu-hawaiians.vercel.app/

Reply:
Yup looks good, the tool tip idea was more for the combined filter categories. I probably should've changed the order of my comments. Update looks good.